### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +18,59 @@ TODO: Add voting-method-http-api specific configuration.
...
+### State voting-methods
+
+To find the ways in which a resident of a state may typically vote there, send
+a GET request to the `/:state` endpoint with any 2-letter state postal
+abbreviation (case insensitive).
+
+On success it will respond with something like:
+
+```clojure
+{:[type] {...}}
```
Comment (danielglauser) [inline README.md]: Could we be a little more specific here?

File: src/voting_method_http_api/handlers.clj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+(ns voting-method-http-api.handlers
+  (:require [clojure.tools.logging :as log]))
+
+(defn ok
```
Comment (danielglauser) [inline src/voting_method_http_api/handlers.clj]: Is this supposed to be `ping` now?

File: test/voting_method_http_api/service_test.clj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+(ns voting-method-http-api.service-test
+  (:require [voting-method-http-api.server :as server]
+            [clj-http.client :as http]
+            [clojure.edn :as edn]
+            [cognitect.transit :as transit]
+            [clojure.core.async :as async]
+            [clojure.test :refer :all]))
+
+(def test-server-port 56000) ; FIXME: Pick a port unique to this project
```
Comment (danielglauser) [inline test/voting_method_http_api/service_test.clj]: Isn't this port unique? What needs to be fixed here?

**Comment by danielglauser** [comment]
Done! Looks good besides the comments I've dropped.

File: README.md (hunk 1 above)
Comment (cap10morgan) [inline README.md]: This isn't even accurate anymore. I'll update it and include an example. The details are highly state-specific, so describing them generically is inherently vague.

File: src/voting_method_http_api/handlers.clj (hunk 2 above)
Comment (cap10morgan) [inline src/voting_method_http_api/handlers.clj]: No, they're `ok` everywhere. But we don't use them for anything regularly.

File: test/voting_method_http_api/service_test.clj (hunk 3 above)
Comment (cap10morgan) [inline test/voting_method_http_api/service_test.clj]: Ah, no, it isn't across projects. I'll give it a different one.

**Comment by cap10morgan** [comment]
@danielglauser How's it look now?

**Comment by danielglauser** [comment]
LGTM - SHIP IT

**Pull request merged**